pkgs/applications/audio/drumkv1/default.nix

```diff
@@ -1,17 +1,52 @@
-{ mkDerivation, lib, fetchurl, pkg-config, libjack2, alsa-lib, libsndfile, liblo, lv2, qt5 }:
+{ 
+  stdenv, lib, pkg-config
+,  fetchurl
+,  cmake
+,  libjack2
+,  alsa-lib
+,  libsndfile
+,  liblo
+,  lv2
+,  qt6
+,  xorg 
+}:
 
-mkDerivation rec {
+stdenv.mkDerivation rec {
   pname = "drumkv1";
-  version = "0.9.23";
+  version = "1.0.0";
 
   src = fetchurl {
     url = "mirror://sourceforge/drumkv1/${pname}-${version}.tar.gz";
-    sha256 = "sha256-gNscsqGpEfU1CNJDlBAzum9M0vzJSm6Wx5b/zhOt+sk=";
+    sha256 = "sha256-vi//84boqaVxC/KCg+HF76vB4Opch02LU4RtbVaxaX4=";
   };
 
-  buildInputs = [ libjack2 alsa-lib libsndfile liblo lv2 qt5.qtbase qt5.qttools ];
+  buildInputs = [ 
+    libjack2 
+    alsa-lib 
+    libsndfile 
+    liblo 
+    lv2 
+    qt6.full 
+    xorg.libX11 
+    qt6.qtbase 
+    qt6.qtwayland 
+  ];
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config cmake qt6.wrapQtAppsHook ];
+
+  cmakeFlags = [ "-DCONFIG_LV2_PORT_CHANGE_REQUEST=false" ]; # disable experimental feature "LV2 port change request"
+
+  installPhase = let
+  lv2Dir = "$out/lib/lv2/drumkv1.lv2";
+  in ''
+    runHook preInstall
+    make install
```

Comment:
We should move everything into postInstall and avoid calling make install by hand.